supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--x.adoc

```diff
@@ -0,0 +1,46 @@
+= X
+
+//Should this term be added to the general conventions "X" module.
+[discrete]
+== XML
+
+*Description* eXtensible Markup Language.  Always format in all caps.
```

Comment:
There are two spaces after the first sentence.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.